crossdock/server/start.go

```diff
@@ -39,4 +39,9 @@ func Start() {
 func Stop() {
 	tch.Stop()
 	yarpc.Stop()
+	http.Stop()
+	apachethrift.Stop()
 }
+
+// TODO(abg): We should probably use defers to ensure things that started up
```

Comment:
I was referring to deferring Stop as each Start is called. Right now, we have:

```
Start = a.Start(); b.Start(); c.Start()  // log.Fatal if anything fails
Stop = a.Stop(); b.Stop(); c.Stop()      // log.Print if anything fails

Start()
defer Stop()
```

We'll call all stop functions even if the first one fails. This will work, but it's not very clean. Ideally we'd schedule the Stop for each server as we Start it. For this, we'd probably have to have a `withServer(func())` function.